app/src/main/java/cm/aptoide/pt/app/AppViewAnalytics.java

```diff
@@ -311,7 +312,11 @@ public void setupDownloadEvents(Download download, int campaignId, String abTest
         DownloadAnalytics.AppContext.APPVIEW, action);
     if (downloadAction == DownloadModel.Action.INSTALL) {
       downloadAnalytics.installClicked(download.getMd5(), download.getPackageName(), trustedValue,
-          editorsChoice, InstallType.INSTALL, action, offerResponseStatus);
+          editorsChoice, InstallType.INSTALL, action, offerResponseStatus, false);
+    }
+    if (downloadAction.equals(DownloadModel.Action.MIGRATE)) {
+      downloadAnalytics.installClicked(download.getMd5(), download.getPackageName(), trustedValue,
```

Comment:
Does it make sense to call the installClicked on Migrate? It looks like we are polluting the installClicked method with migration boolean when it is not exactly an install. if there were a method migrateClicked we could delete that isMigration flag. Logic could be reused in both methods.

Reply:
This is as described in the ticket, as it asked to change this event to add this field